packages/golub/README.md

```diff
@@ -1,19 +1,51 @@
-# Pigeon
+<div align="center">
```

Comment:
file contains lots of references to Pigeon – should be replaced with `golub`

Reply:
done